ui/assets/tad.css

```diff
@@ -0,0 +1,33 @@
+.tad_hidden {
+    display: none;
+}
+
+.tad_visible {
+    display: block;
+}
+
+table.inferenceInput {
+    border: none;
+    margin-bottom: 1em;
+
+}
+
+table.inferenceInput input {
+    height: auto;
+    padding: 0 0 0 0.5em;
+    width: 3em;
+}
+
+table.inferenceInput td, table.inferenceInput th {
+    vertical-align: middle !important;
```

Comment:
Do you really need !important here?

Reply:
With a little more effort, probably not. For now, yes.